src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -29,6 +29,11 @@
 (defn create-section-header [text]
   [:div {:style {:fontSize "125%" :fontWeight 500}} text])
 
+(defn create-inline [& children]
+  [:div {:style {:display "inline" :margin "17px 0 0.33333em 0" :paddingBottom "2.5em"
+                 :fontSize "70%" :lineHeight 1.5  :fontWeight 100}}
+   children])
+
 (defn create-paragraph [& children]
```

Comment:
This create-inline function is so that we can place the "Download Attributes" link and "Import Attributes..." button in the same line as the "Workspace Attributes" header in the summary page.